tests/integration/ha_tests/helpers.py

```diff
@@ -46,33 +52,33 @@ class ProcessRunningError(Exception):
     """Raised when a process is running when it is not expected to be."""
 
 
-def replica_set_client(replica_ips: List[str], password: str, app=APP_NAME) -> MongoClient:
+def replica_set_client(replica_ips: List[str], password: str, app_name=None) -> MongoClient:
     """Generates the replica set URI for multiple IP addresses.
 
     Args:
         replica_ips: list of ips hosting the replica set.
         password: password of database.
-        app: name of application which hosts the cluster.
+        app_name: name of application which hosts the cluster.
     """
     hosts = ["{}:{}".format(replica_ip, PORT) for replica_ip in replica_ips]
     hosts = ",".join(hosts)
 
-    replica_set_uri = f"mongodb://operator:" f"{password}@" f"{hosts}/admin?replicaSet={app}"
+    replica_set_uri = f"mongodb://operator:" f"{password}@" f"{hosts}/admin?replicaSet={app_name}"
     return MongoClient(replica_set_uri)
 
 
-async def fetch_replica_set_members(replica_ips: List[str], ops_test: OpsTest):
+async def fetch_replica_set_members(replica_ips: List[str], ops_test: OpsTest, app_name=None):
```

Comment:
Similar to the above comment, I don't think `app_name` can be `None`

Reply:
agreed, what do you think @dmitry-ratushnyy ?

Reply:
Yes, "None" was added as tmp solution, we can remove it. Thank you 😄